### PR TITLE
New style for Informationswissenschaft: Theorie, Methode und Praxis

### PR DESCRIPTION
**Pull request by nchachereau**
This is a new style created for the journal [Informationswissenschaft: Theorie, Methode und Praxis / Sciences de l’information: théorie, méthode et pratique](https://bop.unibe.ch/iw/), published by the education program in information, library and archival science of the universities of Berne and Lausanne (Switzerland).

I was mandated by the editors of the journal, who will add author guidelines on the website. The style is based on the infoclio.ch style developed for history students in Switzerland (#5754). Changes were made in order for the style to be usable both in French and in German (the journal is bilingual).

Many thanks for all your work!

### CODE REVIEW

**Comment by nchachereau** [comment]
Dear maintainers,

**Is there anything I / we can do to help with getting this new style included in the repository?**

I checked the [Criteria for inclusion](https://github.com/citation-style-language/styles/blob/master/README.md#criteria-for-inclusion) again, and it seems to me that this would fit. A [style guide](https://bop.unibe.ch/iw/about/submissions) has been added to the journal website. 

I had a look again at the [style requirements](https://github.com/citation-style-language/styles/blob/master/STYLE_REQUIREMENTS.md), and it looks like the style should pass all criteria. There would just be a question regarding the file name (and style ID) because the journal has a long bilingual title. Only the German part of the title was used to create the file name, I hope that is okay.

The journal editors, who have commissioned the creation of this CSL style, have asked about the status of the submission. I would be grateful if I could report back.

Thank you so much for your work again!

File: informationswissenschaft-theorie-methode-praxis.csl [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+    </choose>
+  </macro>
+  <macro name="url-or-doi">
+    <choose>
+      <if variable="DOI">
+        <group delimiter=", ">
+          <group delimiter="">
+            <text value="&lt;"/>
+            <text variable="DOI" prefix="https://doi.org/"/>
+            <text value="&gt;"/>
```
Comment (bwiernik) [inline informationswissenschaft-theorie-methode-praxis.csl:533]: Can you move the < and > text values to be part of the prefix and suffix on the DOI? Then you can remove the group here

**Review by bwiernik — COMMENTED** [review]
Thanks for your patience. Can you make the requested change and I'll merge ?

File: informationswissenschaft-theorie-methode-praxis.csl (hunk 1 above)
Comment (nchachereau) [reply informationswissenschaft-theorie-methode-praxis.csl]: Sure, sorry about that!

**Pull request merged**